client/src/main/java/io/pravega/client/netty/impl/ConnectionPoolImpl.java

```diff
@@ -257,22 +256,25 @@ public void initChannel(SocketChannel ch) throws Exception {
     /**
      * Obtain {@link SslContext} based on {@link ClientConfig}.
      */
-    private SslContext getSslContext() {
+    @VisibleForTesting
+    SslContext getSslContext() {
         final SslContext sslCtx;
         if (clientConfig.isEnableTlsToSegmentStore()) {
             log.debug("Setting up an SSL/TLS Context");
             try {
                 SslContextBuilder clientSslCtxBuilder = SslContextBuilder.forClient();
+
                 if (Strings.isNullOrEmpty(clientConfig.getTrustStore())) {
-                    clientSslCtxBuilder = clientSslCtxBuilder.trustManager(FingerprintTrustManagerFactory
-                                                                                   .getInstance(FingerprintTrustManagerFactory.getDefaultAlgorithm()));
-                    log.debug("SslContextBuilder was set to an instance of {}", FingerprintTrustManagerFactory.class);
+                    log.debug("Client truststore wasn't specified.");
+                    File clientTruststore = null; // variable for disambiguating method call
+                    clientSslCtxBuilder.trustManager(clientTruststore);
```

Comment:
This block of setting the null for trustManager is not necessary, but no harm If you want to make it explicit.

Reply:
Yes, I wanted to make it explicit.